lib/protocol/background.js

```diff
@@ -12,6 +12,7 @@
  * @see  https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/appium-bindings.md#background-app
  * @type mobile
  * @for android
+ * @for iOS
```

Comment:
Not sure if the docs parse can interpret this. Please adjust it like: https://github.com/webdriverio/webdriverio/blob/master/lib/protocol/performMultiAction.js#L9

Reply:
I fixed it.
https://github.com/webdriverio/webdriverio/pull/2342/commits/d766145cffc5df49a5780114fd218554a2009e75